scripts/repos.py

```diff
@@ -115,61 +123,74 @@ def print_versions(versions, verbose):
 
     return 0
 
+
 def find_all_git_repositories(checkout_path):
-    """ Walk repositories to find all GIT checkouts."""
-    assert isinstance(checkout_path, str), "Expecting a string for the directory name."
+    """Walk repositories to find all GIT checkouts."""
+    assert isinstance(checkout_path,
+                      str), "Expecting a string for the directory name."
     if not os.path.isdir(checkout_path):
-        die("repositories location '{}' does not exists !".format(checkout_path))
+        die("repositories location '{}' does not exists !".format(
+            checkout_path))
     repos = list()
     for root, sub_dirs, _ in os.walk(checkout_path):
         if '.git' in sub_dirs:
             repos.append(root)
     return repos
 
+
 def get_repositories_status(checkout_path):
-    """ Get the state of each git repository."""
-    assert isinstance(checkout_path, str), "Expecting a string for the directory name."
+    """Get the state of each git repository."""
+    assert isinstance(checkout_path,
+                      str), "Expecting a string for the directory name."
     status = dict()
     repos = find_all_git_repositories(checkout_path)
     rex = re.compile('^' + checkout_path + r'\/')
     for repo_path in repos:
         repo = git.Repo(repo_path)
         rel_path = rex.sub('', repo_path)
         status[rel_path] = {
-                       'SHA1': repo.head.commit.hexsha,
-                       'Dirty': repo.is_dirty(),
-                       'URL': repo.git.config('--get', 'remote.origin.url')
-                     }
+            'SHA1': repo.head.commit.hexsha,
+            'Dirty': repo.is_dirty(),
+            'URL': repo.git.config('--get', 'remote.origin.url')
+        }
         if repo.head.is_detached:
             status[rel_path]['Branch'] = None
         else:
             status[rel_path]['Branch'] = repo.active_branch.name
     return status
 
+
 def print_repositories_status(repositories):
-    """ Report the state of each git repository."""
-    assert isinstance(repositories, str), "Expecting a string for the directory name."
+    """Report the state of each git repository."""
+    assert isinstance(repositories,
+                      str), "Expecting a string for the directory name."
```

Comment:
Urgh! I see that it gets the line length shorter, but I'm not sure it actually improves readability. How about
```
assert isinstance(repositories, str), (
    "Expecting a string for the directory name.")
```

Reply:
I've updated the patch series. I decided to leave the white space formatting part as-is because some lines (including most assert statements) are removed in the third patch. The third patch also deals with formatting issues left after the tool.